security/tor/src/opnsense/mvc/app/models/OPNsense/Tor/General.xml

```diff
@@ -99,5 +99,23 @@
       <default>0</default>
       <Required>Y</Required>
     </dns_map_hosts>
+    <client_authentications>
+      <client_auth type="ArrayField">
+        <enabled type="BooleanField">
+          <default>1</default>
+          <Required>Y</Required>
+        </enabled>
+        <onion_service type="TextField">
+          <Required>Y</Required>
+          <default>exampleexample23.onion</default>
+          <mask>/^[a-z2-7]{16}\.onion$/i</mask>
+        </onion_service>
+        <auth_cookie type="TextField">
+          <Required>Y</Required>
+          <default>0</default>
```

Comment:
just for your information - this is stupid too

Reply:
:D